### DISCUSSION TITLE
Skywalk-client-js0.10.0 seems to have forgotten build

### DISCUSSION THREAD

**Discussion by baixjian** [apache/skywalking#11110] [General]
<img width="1529" alt="image" src="https://github.com/apache/skywalking/assets/109275788/7d9b1ef4-96df-4c8a-8d94-d89854209f4e">
but when i download  https://github.com/apache/skywalking-client-js/releases/tag/v0.10.0
Nothing seems to have changed

**Comment by Fine0830** [comment]
https://github.com/apache/skywalking-client-js/pull/114 is tagged as `0.11.0`.

**Reply by wu-sheng** [reply]
@Fine0830 Is that a release bug or unreleased thing?
I am confused about the description.

**Comment by Fine0830** [comment]
It hasn't been released yet.

**Reply by baixjian** [reply]
I appreciate your contributions，If possible, I would like to know when it will be released
